packages/elements/src/components/Tabs/index.tsx

```diff
@@ -9,18 +10,19 @@ export interface TabConfig {
 
 export interface TabsProps {
   tabConfigs: TabConfig[]
+  className?: string
 }
 
 export const selectTab = (event: React.SyntheticEvent, tabConfig: TabConfig) => {
   event.preventDefault()
   tabConfig.onTabClick(tabConfig.tabIdentifier)
 }
 
-export const Tabs: React.FunctionComponent<TabsProps> = ({ tabConfigs }) => (
-  <div className="tabs is-fullwidth" role="tablist">
+export const Tabs: React.FunctionComponent<TabsProps> = ({ tabConfigs, className }) => (
+  <div className={cx(className, 'tabs', 'is-fullwidth')} role="tablist">
```

Comment:
we should move `className` to the last position
cx('tabs', 'is-fullwidth', className)

Reply:
fixed